.changeset/giant-kids-promise.md

```diff
@@ -0,0 +1,5 @@
+---
+"@t3-oss/env-core": minor
```

Comment:
Someone got mad earlier that I released a 0.x minor when there were no breaking changes so let's keep it patch this time 😅 

```suggestion
"@t3-oss/env-core": patch
```